CALEngine/src/bmi.cc

```diff
@@ -225,3 +225,7 @@ vector<pair<uint32_t, float>> BMI::get_top_terms(string doc_id, int num_top_term
 unique_ptr<BMI::StratumInfo> BMI::get_stratum_info(){
     return nullptr;
 } 
+
+std::vector<std::pair<std::string, float>> BMI::get_stratum_docs() {
```

Comment:
Can you allow this function to specify the stratum number. It seems to be only returning the last stratum docs.

Reply:
Improved